platform/ios/vendor/SMCalloutView/SMCalloutView.m

```diff
@@ -62,6 +64,28 @@ - (id)initWithFrame:(CGRect)frame {
     return self;
 }
 
+#ifdef CHECK_CALLOUT_SHAPE_FOR_HIT_TEST
+- (UIView*)hitTest:(CGPoint)point withEvent:(UIEvent *)event {
+    UIView* hitView = [super hitTest:point withEvent:event];
```

Comment:
Super-pedantic-nit: the asterisk in `UIView *` is attached to the wrong side on this line and the one above.